frontend/packages/console-shared/src/components/quick-search/QuickSearchModalBody.tsx

```diff
@@ -44,7 +44,7 @@ const QuickSearchModalBody: React.FC<QuickSearchModalBodyProps> = ({
   viewContainer,
 }) => {
   const DEFAULT_HEIGHT_WITH_NO_ITEMS = 60;
-  const DEFAULT_HEIGHT_WITH_ITEMS = 400;
+  const DEFAULT_HEIGHT_WITH_ITEMS = 483;
```

Comment:
I suspect this change is not really going to be a thing we can chase. We should definitely have a dynamic one.

Eg:
![Screen Shot 2021-08-31 at 11 50 44 AM](https://user-images.githubusercontent.com/8126518/131535135-2393d805-977b-4048-b903-54e5c0f1c040.png)
![Screen Shot 2021-08-31 at 11 50 56 AM](https://user-images.githubusercontent.com/8126518/131535140-d7cf29c7-5e02-487e-8eef-592e72205e0b.png)